.eslintrc

```diff
@@ -76,7 +76,7 @@
         "no-shadow": 2,
         "no-undef": 2,
         "no-unused-vars": 0,
-        "no-use-before-define": 1,
+        "no-use-before-define": ["error", { "functions": false }],
```

Comment:
not a huge fan of this change but I'll let it in but we need to change that. 
it is not best practice to use a function before it is defined.